apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -308,6 +308,13 @@ static CareTeamComponent addCareTeamPractitioner(
                 ctc ->
                     practitionerIdSystem.equals(ctc.getProvider().getIdentifier().getSystem())
                         && practitionerIdValue.equals(ctc.getProvider().getIdentifier().getValue()))
+            .filter(ctc -> ctc.hasRole())
```

Comment:
Previously, this statement would only consider NPI number. This would prevent the same NPI from being used for different roles.